app/src/main/java/com/example/sabonit/Account.java

```diff
@@ -13,29 +13,17 @@
  */
 public class Account
 {
-    /* ********* Constants: ********* */
-    // For validity phone number checking
-    private static int IMPLICIT_ISRAELI_PHONE_NUMBER_LENGTH = 9;
-    private static int ISRAELI_AREA_CODE = 13;
-    private static String ISRAELI_AREA_CODE_INFIX = "+972";
-
 
     /* ********* Attributes: ********* */
-    // Given as an address in the next pattern: street and building number, city ("Ramban 30, Jerusalem")
-    private String location;
-    // User's name, may can be extracted from the google acount
+    // User's name, may can be extracted from the google account
     private String name;
-    // User's phone number, may can be extracted from the google acount
-    private String phoneNumber;
     // User's cart, empty when initialized
     private Cart cart;
+    //User's UID of the google account, unique value
+    private String UID;
     // The account of the user that is logged in
     private static Account currentAccount;
```

Comment:
i wonder if this solution for the access to currentAccount is a good one...
static variables are related to the class itself, not to the instant.
did you think about the option of transfer this data through the intant (i mean, between the activities)?

Reply:
In my opinion the solutions are equal. ma gam that I talked with Moshe and Alex and there is no preference for one of them.
just notice that this was the original solution of gal, not mine, i just didnt change it now